pkg/osquery/table/touchid_system_darwin.go

```diff
@@ -41,7 +41,10 @@ func (t *touchIDSystemConfigTable) generate(ctx context.Context, queryContext ta
 
 	// Read the security chip from system_profiler
 	var stdout bytes.Buffer
-	cmd := exec.CommandContext(ctx, "/usr/sbin/system_profiler", "SPiBridgeDataType")
+	cmd, err := allowedcmd.Systemprofiler(ctx, "SPiBridgeDataType")
+	if err != nil {
+		return nil, fmt.Errorf("creating system_profiler command: %w", err)
```

Comment:
This is a table -- it'll be better to log the error and return no rows. (Though I see from a couple lines down, that's not what we've done prior)

Reply:
I'm happy to update -- I'll go through and review anywhere I've touched a table to make sure behavior is consistent!

Reply:
This is actually the only place with that issue -- I've updated.